include/abb_librws/rws_client.h

```diff
@@ -385,6 +385,24 @@ class RWSClient : public POCOClient
    */
   RWSResult getIOSignal(const std::string& iosignal);
 
+  /**
+   * \brief A method for retrieving static information about a mechanical unit.
+   *
+   * \param mechunit for the mechanical unit's name.
+   *
+   * \return RWSResult containing the result.
```

Comment:
```suggestion
   * \return RWSResult the result.
```

But even the original comment doesn't really add much. It's "the result", but that was quite clear from the name of the type ;)

Reply:
Same comment as above